app/routes/companyInterest/components/CompanyInterestPage.tsx

```diff
@@ -228,7 +227,7 @@ const LanguageFlag = ({ language }: { language: string }) => {
       break;
   }
 
-  return <Image src={flag} className={styles.flag} alt="country_flag" />;
+  return <img src={flag} className={styles.flag} alt="Country flag" />;
```

Comment:
Why no `Image`?

Reply:
Because the `Image` component comes with a `width: 100%`, which overrides the styling given by `styles.flag`. Besides, there's no real reason to use the `Image` component here because we don't utilize any of its features, like progressive or lazy loading, since the images are loaded directly from the assets.